docs/source/reference/bentoml/bento-build-options.rst

```diff
@@ -0,0 +1,557 @@
+===================
+Bento build options
+===================
+
+Build options refer to a set of configurations defined in a YAML file (typically named ``bentofile.yaml``) for building a BentoML project into a Bento.
```

Comment:
Can you also mention that `pyproject.toml` is also supported, the configurations should go under `[tool.bentoml.build]` section, and the fields are the same.

Ideally, display a tab component for each config snippet, with `pyproject.toml` and `bentofile.yaml` side-by-side. The former is preferred.

Reply:
Added.